go/test/endtoend/transaction/twopc/schema.sql

```diff
@@ -9,4 +9,18 @@ create table twopc_music (
     user_id bigint,
     title varchar(64),
     primary key (id)
-) Engine=InnoDB;
\ No newline at end of file
+) Engine=InnoDB;
+
+create table twopc_fuzzer_update (
+    id bigint,
+    col bigint,
+    primary key (id)
+) Engine=InnoDB;
+
+create table twopc_fuzzer_insert (
+    id bigint,
+    updateSet varchar(50),
```

Comment:
should `updateSet` be number?

Reply:
I added shard information + update set number and then stored it as a string. I can split into 2 int columns, but didn't feel the need right now.

Reply:
I'm changing this to be simpler to read.

Reply:
Fixed